src/lib.rs

```diff
@@ -159,6 +159,15 @@ pub enum FillInReduction {
     ReverseCuthillMcKee,
 }
 
+#[cfg(feature = "approx")]
+/// Traits for comparing vectors and matrices using the approx traits
+///
+/// Comparisons of sparse matrices with different matrices might be slow.
```

Comment:
> Comparisons of sparse matrices with different matrices might be slow.
I think you meant "with different storages" and not "with different matrices"

Reply:
Indeed!